WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -440,6 +440,7 @@ class AztecPostViewController: UIViewController, PostEditor {
 
         PostCoordinator.shared.cancelAnyPendingSaveOf(post: post)
         addObservers(toPost: post)
+        registerMediaObserver()
```

Comment:
The observer is currently set when the post is set (which happens a few lines above this) – could this result in multiple observers being registered?

Reply:
Hmm, in testing this seems fine. I think the didSet post one doesn't get called on initialisation?

Reply:
> doesn't get called on initialisation?

Exactly, here is the relevant documentation: 

```The willSet and didSet observers of superclass properties are called when a property is set in a subclass initializer, after the superclass initializer has been called. They are not called while a class is setting its own properties, before the superclass initializer has been called.

For more information about initializer delegation, see Initializer Delegation for Value Types and Initializer Delegation for Class Types.```